### PR TITLE
Fix jQuery selector

### LINKED ISSUE #189: Drug testing checkbox
Received an email from Dave K about this today:

> Supervisors are running into problems when attempting to check off which drug testing threshold was met (bodily injury / disabling damage / fatality since they are not prompted to select one after indicating that a post-accident test was conducted. We currently have to check threshold boxes off by hand before scanning the final product in if they do not appear as an option for the supervisor.

### PR DESCRIPTION
**Pull request by werebus**
To allow editing the drug test "reason" fields. Closes #189

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ gem 'haml-rails'
 gem 'jbuilder', '~> 2.5'
 gem 'mysql2'
 gem 'nokogiri', '>= 1.8.1'
+gem 'openssl'
```
Comment (Anbranin) [inline Gemfile]: Just curious why this change

Comment (werebus) [reply Gemfile]: Because it told me to? I actually cna't reproduce the problem anymore, but in Ruby 2.5, a number of the standard libraries were moved to "default gems". That is, they're gems, but gems that come with ruby. I've also had this problem crop up from time to time with `CSV`.

https://www.slideshare.net/hsbt/gemification-for-ruby-2530

Comment (Anbranin) [reply Gemfile]: Ohhh okay.
